packages/attribute-registry-consumer/src/model/converter.ts

```diff
@@ -14,7 +14,9 @@ export const fromAttributeKind = (
     .with(AttributeKindV1.CERTIFIED, () => persistentAttributeKind.certified)
     .with(AttributeKindV1.DECLARED, () => persistentAttributeKind.declared)
     .with(AttributeKindV1.VERIFIED, () => persistentAttributeKind.verified)
-    .otherwise(() => persistentAttributeKind.declared); // TODO check for exhaustive match
+    .otherwise(() => {
```

Comment:
@galales @beetlecrunch We can't use `exhaustive()` here (As of this [issue](https://github.com/gvergnaud/ts-pattern/issues/183)) so we thought to handle it like this.

Let us know if you have any suggestions on how to handle this case.